src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -629,6 +665,7 @@ private void AddTypeWithLoopCheck(IGraphType resolvedType, TypeCollectionContext
             context.InFlightRegisteredTypes.Push(namedType);
             try
             {
+                using var _ = context.Trace($"AddTypeWithLoopCheck for type '{namedType.Name}'");
```

Comment:
I would like to point out that these strings get created whether tracing is enabled or not.  Either we can leave it, which is fine, or we can add the name as a second argument, like so:

```cs
IDisposable Trace(string description, string? param = null)
{
    if (tracing is enabled)
        description = param == null ? description : $"{description} '{param}'";
    // other code
}

using var _ = context.Trace("AddTypeWithLoopCheck for type", namedType.Name);
```

Reply:
(obviously this is just an example)

Reply:
I know, can change it.

Reply:
If you plan to do so, then I'll review again after you do so.  Everything else looks fine.

Reply:
done